### PR TITLE
Update PGP key servers

### PR DESCRIPTION
**Pull request by tvoran**
Use hkps://keys.openpgp.org and hkps://keyserver.ubuntu.com as
recommended in the [docker-library faq](https://github.com/docker-library/faq#openpgp--gnupg-keys-and-verification).

Related to https://github.com/docker-library/faq/pull/26

### CODE REVIEW

**Review by calvn — APPROVED** [review]
Makes sense, thanks for updating!

**Comment by kalafut** [comment]
This also closes #213 I think.

**Review by benashz — APPROVED** [review]
Suggest to use TLS for all GPG key servers.

File: ubi/Dockerfile [hunk 1]
```diff
@@ -22,8 +22,8 @@ RUN set -eux; \
     VAULT_GPGKEY=C874011F0AB405110D02105534365D9472D7468F; \
     found=''; \
     for server in \
-        hkp://p80.pool.sks-keyservers.net:80 \
-        hkp://keyserver.ubuntu.com:80 \
+        hkps://keys.openpgp.org \
+        hkps://keyserver.ubuntu.com \
         hkp://pgp.mit.edu:80 \
```
[the remark this answers is not in the document]
Comment (tvoran) [reply ubi/Dockerfile]: Sure, that makes sense.

**Pull request merged**